modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/NuxeoException.java

```diff
@@ -99,11 +99,12 @@ public NuxeoException(Throwable cause, int statusCode) {
      * @param info the information
      * @since 7.4
      */
-    public void addInfo(String info) {
+    public NuxeoException addInfo(String info) {
```

Comment:
FYI this is not binary compatible so if this has to be backported as is I'd use a new API (like `withInfo`).
Otherwise don't backport this part and just call `addInfo` in a separate statement.

Reply:
Indeed, I won't backport it to 10.10 and leave the method as is on master.